charts/sn-platform/values.yaml

```diff
@@ -825,6 +825,9 @@ bookkeeper:
     # use a persistent volume or emptyDir
     persistence: true
     journal:
+      # It determines the directory of journal data
+      numVolumes: 1
+      numDirsPerVolume: 1
```

Comment:
`numDirsPerVolume` is 1 means that there can be only one directory on a volume, is that correct? @fantapsody